### PR TITLE
[Round 3 Method Reference]Lps 181270

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-181271

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:ldap

File: modules/apps/portal-settings/portal-settings-authentication-ldap-web/src/main/java/com/liferay/portal/settings/authentication/ldap/web/internal/portlet/action/EditLDAPServerMVCActionCommand.java [hunk 1]
```diff
@@ -302,18 +256,34 @@ private void _validateSearchFilters(ActionRequest actionRequest)
...
+
+	@Reference
 	private Portal _portal;
+
+	@Reference(
+		target = "(javax.portlet.name=" + ConfigurationAdminPortletKeys.INSTANCE_SETTINGS + ")"
+	)
 	private Portlet _portlet;
+
 	private PortletContext _portletContext;
```
Comment (dantewang) [inline modules/apps/portal-settings/portal-settings-authentication-ldap-web/src/main/java/com/liferay/portal/settings/authentication/ldap/web/internal/portlet/action/EditLDAPServerMVCActionCommand.java:279]: @slnn This was added by LPS-61991, but the usage was soon removed by LPS-67555. Please remove this field, `_portletContextFactory`, and other fields if possible.

**Comment by dantewang** [comment]
@slnn Please see my comment and resend, thanks.

**Comment by slnn** [comment]
Please see https://github.com/liferay-core-infra/liferay-portal/pull/4233

**Pull request closed without merging**